tests/operators/functionals/test_l2_squared_norm.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for L2-Squared-functional."""
+
+# Copyright 20234 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import pytest
+import torch
+from mrpro.operators.functionals.l2_squared import L2NormSquared
+
+
+@pytest.mark.parametrize(
+    (
+        'x',
+        'expected_result_x',
+        'expected_result_p',
+        'expected_result_pcc',
+        'expected_result_p_forward',
+        'expected_result_pcc_forward',
+    ),
+    [
+        (
+            torch.tensor([1, 1], dtype=torch.complex64),
```

Comment:
where do these come from?
did you check that these are the expected results for proximal and proximal of convex conjugate?
... just in case: using the outputs of the function as expected results does not really help